src/library.rs

```diff
@@ -1006,7 +1013,7 @@ impl Library {
                 let full_name = format!("{}.{}", namespace_name, name);
                 let mut check_methods = true;
 
-                if !not_allowed_ending.iter().any(|s| name.ends_with(s)) {
+                if !not_allowed_ending.iter().any(|s| name.ends_with(s)) || x.is_enumeration() {
```

Comment:
Also flags I guess?

Reply:
flags?

Reply:
Aka `Bitfield`, the thing we use the `bitflags` crate for

Reply:
Oh I see! Added it. :)